cli/src/tasks/copy.ts

```diff
@@ -70,23 +72,55 @@ export async function copy(
       'capacitor:copy:before',
     );
 
+    const allPlugins = await getPlugins(config, platformName);
+    let usesCapacitorPortals = false;
+    if (
+      allPlugins.filter(plugin => plugin.id === '@capacitor/federation')
```

Comment:
This needs to be changed to reflect the actual package name we use for Capacitor Portals Plugin